framework/src/application/node/forger/forger.ts

```diff
@@ -86,26 +91,26 @@ interface ForgerConstructor {
 }
 
 export class Forger {
-	private readonly logger: Logger;
-	private readonly storage: Storage;
-	private readonly processorModule: ProcessorModule;
-	private readonly dposModule: Dpos;
-	private readonly bftModule: BFT;
-	private readonly transactionPoolModule: TransactionPool;
-	private readonly chainModule: Chain;
-	private readonly keypairs: KeyPairs;
-	private readonly config: {
+	private readonly _logger: Logger;
+	private readonly _storage: Storage;
+	private readonly _processorModule: ProcessorModule;
+	private readonly _dposModule: Dpos;
+	private readonly _bftModule: BFT;
+	private readonly _transactionPoolModule: TransactionPool;
+	private readonly _chainModule: Chain;
+	private readonly _keypairs: KeyPairs;
+	private readonly _config: {
 		readonly forging: {
```

Comment:
I missed this out before but why we don't have any access modifier for this property? `public`?

Reply:
_config is set to private so.

Reply:
but the property should have an access modifier for consistency? right now someone can expose it if it doesn't need to be exposed

Reply:
Sorry I didn't see that it was nested, all good